Graphics/ShaderTools/CMakeLists.txt

```diff
@@ -43,7 +43,11 @@ if((PLATFORM_WIN32 AND NOT MINGW_BUILD) OR PLATFORM_UNIVERSAL_WINDOWS OR PLATFOR
     set(DXC_SUPPORTED TRUE)
 endif()
 
-if (DXC_SUPPORTED)
+if(NOT DILIGENT_NO_DIRECT3D11 OR NOT DILIGENT_NO_DIRECT3D12)
+  set(DXC_NEEDED TRUE)
+endif()
+
+if (DXC_SUPPORTED AND DXC_NEEDED)
```

Comment:
This is also incorrect. DXC is used by Vulkan backend even when there is no D3D11 and D3D12. All high-end tutorials (mesh shaders, ray tracing, hybrid rendering) require DXC compiler in Vulkan mode